electron-builder.config.cjs

```diff
@@ -75,10 +75,7 @@ const config = {
     win: {
         icon: "build/icons.icns",
         publisherName: "Command Line Inc",
-        target: ["nsis", "msi", "zip"],
-        certificateSubjectName: "Command Line Inc",
-        certificateSha1: process.env.SM_CODE_SIGNING_CERT_SHA1_HASH,
-        signingHashAlgorithms: ["sha256"],
```

Comment:
This is obviously not ideal.  I'm having trouble finding another way around this? we can make a whole new `config.cjs` but that would require maintaining two configs. Any ideas?